src/routes/load/container/Load.tsx

```diff
@@ -37,15 +37,29 @@ export const loadSafe = async (
   await addSafe(safeProps)
 }
 
-class Load extends React.Component<any> {
-  onLoadSafeSubmit = async (values) => {
+interface ILoad {
+  addSafe: Dispatch<any>
+  network: string
+  provider?: string
+  userAddress: string
+}
+
+export interface LoadFormValues {
```

Comment:
1. One of the interfaces is prefixed with I, one is not. Please stay consistent
2. In general there's no need to prefix an interface with `I` in TypeScript, so I wouldn't do that (and we weren't doing that, so I would stay consistent)